src/iidm/Bus.cpp

```diff
@@ -155,6 +155,10 @@ bool Bus::isInMainSynchronousComponent() const {
     return static_cast<bool>(sc) && (sc.get().getNum() == ComponentConstants::MAIN_NUM);
 }
 
+void Bus::visitConnectedEquipments(TopologyVisitor& visitor) {
+    visitEquipments<Terminal>(getConnectedTerminals(), visitor);
```

Comment:
What is better: to have a template method, or to map Node/Bus Terminals into Terminal using a map operator

Reply:
Agree : using an upcast >>> hxx file...